mmcv/ops/csrc/pytorch/npu/bbox_overlaps_npu.cpp

```diff
@@ -22,7 +26,7 @@ void bbox_overlaps_npu(const Tensor bboxes1, const Tensor bboxes2, Tensor ious,
       .Input(gtboxes)
       .Output(ious)
       .Attr("mode", modeStr)
-      .Attr("eps", (float)offset)
+      .Attr("eps", offset_)
```

Comment:
Hi, why is `eps (offset)` used as the denominator?

Reply:
Why is it set as `0.01` rather than other value?

Reply:
This is due to the limitation of the development mode and model scenario. 0.01 and 1 need to be enumerated.